lib/file_scope.rb

```diff
@@ -16,6 +16,13 @@
 # you may find current contact information at www.suse.com
 
 class FileScope < Machinery::Object
+  def write_files_tarball(destination)
+    FileUtils.cp(
+      File.join(scope_file_store.path, "files.tgz"),
+      destination
+    )
+  end
```

Comment:
Maybe the utils class should have a broader scope and not only handle one file, but also the collection of files.